aten/src/ATen/cuda/CUDAGraph.cpp

```diff
@@ -110,9 +109,10 @@ void CUDAGraph::capture_begin(MempoolId_t pool/*=0*/, cudaStreamCaptureMode capt
   AT_CUDA_CHECK(cudaStreamBeginCapture(capture_stream_, capture_mode));
 
   cudaStreamCaptureStatus status{};
-  AT_CUDA_CHECK(cudaStreamGetCaptureInfo(stream, &status, &capture_id_));
+  AT_CUDA_CHECK(cudaStreamGetCaptureInfo_v2(stream, &status, &capture_id_, &graph_, nullptr, nullptr));
```

Comment:
why do we need `graph_` here? To be able to query graph before captureEnd has been called?